openshift/template.yaml

```diff
@@ -233,5 +233,5 @@ objects:
           name: cad-ocm-client-secret
       - secretRef:
           name: cad-pd-token
-      image: ${REGISTRY_IMG}@${IMAGE_DIGEST}
+      image: ${REGISTRY_IMG}:${IMAGE_TAG}
```

Comment:
should be replaced in the `parameters` section as well

Reply:
fixed